parser.js

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env node --abort-on-uncaught-exception
+
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2024 MNX Cloud, Inc.
+ */
+
+
+var child_process = require('child_process');
+var os = require('os');
+var process = require('process');
+
+var VALID_PROTOCOLS = [
+    'tcp',
+    'http',
+    'https'
+];
+
+var MAX_PORT = 65534;
+var MIN_PORT = 1;
+var MAX_BACKENDS_LOW = 32;
+var MAX_BACKENDS_HIGH = 1024;
+var STICKY_MODES = [
+    'http',
+    'https'
+]
```

Comment:
For consistency, should there be a semicolon here?

Reply:
Fixed.